### PR TITLE
Fixed typo in source file linking Sphinx extension.

### PR DESCRIPTION
**Pull request by michaelg100**
# Trac ticket number
"N/A"

# Branch description
Just fixing a few spelling mistakes

# Checklist
- [ ] This PR targets the `main` branch. 
- [ ] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [ ] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: docs/ref/csrf.txt [hunk 1]
```diff
@@ -79,7 +79,7 @@ The CSRF protection is based on the following things:
    requests from ``www.example.com`` and ``api.example.com``. If the setting is
    not set, then the referer must match the HTTP ``Host`` header.
 
-   Expanding the accepted referers beyond the current host or cookie domain can
+   Expanding the accepted referrers beyond the current host or cookie domain can
```
Comment (felixxm) [inline docs/ref/csrf.txt:82]: This is not a typo, check out https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer.

Comment (michaelg100) [reply docs/ref/csrf.txt]: gotcha and saw in the spelling wordlist referers was there too just wasn't sure in this text file

Comment (michaelg100) [reply docs/ref/csrf.txt]: Removed this then

File: docs/_ext/github_links.py [hunk 2]
```diff
@@ -143,7 +143,7 @@ def github_linkcode_resolve(domain, info, *, version, next_version):
 
     branch = get_branch(version=version, next_version=next_version)
     relative_path = path.relative_to(pathlib.Path(__file__).parents[2])
-    # Use "/" explicitely to join the path parts since str(file), on Windows,
+    # Use "/" explicitly to join the path parts since str(file), on Windows,
```
Comment (shangxiao) [inline docs/_ext/github_links.py:146]: 👏

**Review by shangxiao — APPROVED** [review]
🏆

**Review by nessita — APPROVED** [review]
Thank you, great catch!

**Pull request merged**